themes/scss/vendor/leaflet.scss

```diff
@@ -278,12 +278,12 @@
 	border-radius: 5px;
 	}
 .leaflet-control-layers-toggle {
-	background-image: url(images/layers.png);
+	background-image: url(../../img/layers.png);

```

Comment:
Line should be indented with spaces, not tabs<br>Properties should be ordered width, height, background-image<br>URLs should be enclosed in quotes